### ISSUE TITLE
EmscriptenWasm hidden from typescript?

### ISSUE THREAD

**Issue by toloudis** [manzt/numcodecs.js#33]
Hi! I'm trying to use zarr.js (via @loaders.gl/zarr) and in my type check build step, I am getting this error:

```
> tsc -p tsconfig.json --noEmit

node_modules/numcodecs/utils.d.ts:12:56 - error TS2503: Cannot find namespace 'EmscriptenWasm'.

12 export declare function initEmscriptenModule<M extends EmscriptenWasm.Module>(moduleFactory: EmscriptenWasm.ModuleFactory<M>, src: string): Promise<M>;
                                                          ~~~~~~~~~~~~~~

node_modules/numcodecs/utils.d.ts:12:94 - error TS2503: Cannot find namespace 'EmscriptenWasm'.

12 export declare function initEmscriptenModule<M extends EmscriptenWasm.Module>(moduleFactory: EmscriptenWasm.ModuleFactory<M>, src: string): Promise<M>;
                                                                                                ~~~~~~~~~~~~~~


Found 2 errors.

```

This is not preventing my code from running but I'm wondering if you have seen this -- possibly just need to import it in that utils file?

**Comment by manzt** [comment]
@toloudis thanks for opening the issue. I haven't seen this before, but I believe this is because `EmscriptenWasm` types are declared globally in this repo (in `./types/emscripten.d.ts`) and not imported as types from a module (so they don't end up being emitted).

I have opened a PR to address this issue #33 (

TL;DR you should be able to install and include `@types/emscripten` to your tsconfig, or you can use `--skipLibCheck` to avoid type-checking declaration files.

**Comment by toloudis** [comment]
Thank you! For now, skipLibCheck should do the job for me.  I'll update when the PR change is released :)

**Comment by manzt** [comment]
@toloudis Sorry for the delay, but I just made a release. I was having some issues upgrading our test suite in zarr.js https://github.com/gzuidhof/zarr.js/pull/105, but this seems to be resolved. I think you should be able to install with a fresh lock and get the latest version. Let me know if you need something else!
